src/DynamoCoreWpf/Views/Preview/PreviewControl.xaml.cs

```diff
@@ -181,6 +178,8 @@ internal void BindToDataSource(MirrorData mirrorData)
             {
                 RefreshExpandedDisplay(delegate { BeginViewSizeTransition(ComputeLargeContentSize()); });
             }
+
+            IsDataBound = true;
```

Comment:
Yep. It actually could be removed at all. I just keep it there so that it could have same behavior for hidden state.